### ISSUE TITLE
Security: Installer vulnerable to DLL hijacking

### ISSUE THREAD

**Issue by ericlaw1979** [hexchat/hexchat#1622]
The HexChat installer is vulnerable to DLL hijacking as described here: https://textplain.wordpress.com/2015/12/18/dll-hijacking-just-wont-die/

The installer _appears_ to be built on a version of NSIS earlier than v2.5; if that's the case, the fix is pretty simple-- just upgrade to the later build.

**Comment by TingPing** [comment]
> The installer appears to be built on a version of NSIS earlier than v2.5; if that's the case, the fix is pretty simple-- just upgrade to the later build.

No.. it uses Inno 5.5.x. Your link mentions [Inno was vulnerable](http://seclists.org/fulldisclosure/2015/Dec/33) and it hasn't been mentioned in any changelogs. So unless you know something more this is just an upstream issue.

**Comment by TingPing** [comment]
Actually seems it is fixed in upcoming 5.5.9 release: https://github.com/jrsoftware/issrc/issues/188
